src/current/v24.3/create-logical-replication-stream.md

```diff
@@ -0,0 +1,148 @@
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+
+Logical data replication is only supported in CockroachDB {{ site.data.products.core }} clusters.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v24.3" %} The `CREATE LOGICAL REPLICATION STREAM` statement starts [**logical data replication (LDR)**]({% link {{ page.version.version }}/logical-data-replication-overview.md %}) that runs between a source and destination cluster in an active-active setup.
+
+This page is a reference for the `CREATE LOGICAL REPLICATION STREAM` SQL statement, which includes information on its parameters and possible options. For a step-by-step guide to set up LDR, refer to the [Set Up Logical Data Replication]({% link {{ page.version.version }}/set-up-logical-data-replication.md %}) page.
+
+## Required privileges
+
+`CREATE LOGICAL REPLICATION STREAM` requires one of the following privileges:
+
+- The [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role).
+- The [`REPLICATION` system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges).
+
+Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SYSTEM REPLICATION TO user;
+~~~
+
+## Synopsis
+
+<div>
+{% include {{ page.version.version }}/ldr/create_logical_replication_stream_stmt.html %}
+</div>
+
+### Parameters
+
+Parameter | Description
+----------+------------
+`db_object_name` | The fully qualified name of the table on the source or destination cluster. Refer to [Examples](#examples).
+`logical_replication_resources_list` | A list of the fully qualified table names on the source or destination cluster to include in the LDR stream. Refer to the [LDR with multiple tables](#multiple-tables) example.
+`source_connection_string` | The connection string to the source cluster. Use an [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) to store the source cluster's connection URI. To start LDR, you run `CREATE LOGICAL REPLICATION STREAM` from the destination cluster.
+`logical_replication_options` | Options to modify the behavior of the LDR stream.
+
+## Options
+
+Option | Description
+-------+------------
+`cursor` | Emits any changes after the specified timestamp. LDR will not perform an initial backfill with the `cursor` option, it will stream any changes after the specified timestamp. The LDR job will encounter an error if you specify a `cursor` timestamp that is before the configured [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) window for that table. Use `cursor` to [Bootstrap LDR from a backup](#bootstrap-ldr-from-a-backup) to avoid the initial backfill of data onto the destination cluster from source cluster.
```

Comment:
i wonder if we need some warning around the `cursor` option. like, this is a very powerful tool and if the user uses the wrong timestamp, they experience data loss.

Reply:
There is probably a way to add something here to that effect. I've given it a shot PTAL cc @alicia-l2